packages/accounts-2fa/package.js

```diff
@@ -0,0 +1,27 @@
+Package.describe({
+  name: 'accounts-2fa',
+  version: '1.0.0',
+  summary: 'Package used to enable two factor authentication through OTP protocol',
+  git: '',
```

Comment:
We could provide a directly URL to here or remove it.

Reply:
Solved